README.md

```diff
@@ -1 +1,62 @@
-# stats-data
\ No newline at end of file
+# stats-data
+
+## Repo Overview
+
+This repo contains scripts to scrape various community data points.
+
+## Usage
+
+### Requirements
+
+| Script / Service | Requirements |
+| --- | --- |
+| discord | None |
+| github | None |
+| google | Custom Search Engine |
+| reddit | Registered reddit bot |
+| stackexchange | None |
+| twitter | Valid account |
+| youtube | Valid API Key |
+### Configuration
+
+Example `.env`
+
+```
+# Required
+CSE_API_KEY='m3DJU4hokX9haNZjvUWKkYnYUTrujAVD3FgWa9e'
+CSE_CX='dezZ22vMRorVt3eVa'
+TWITTER_AUTHORIZATION_HEADER='Bearer WXeXLCEECWy7hxVVXqA4gUybnzAWgj7gMfaFVnZhWvFDkWRU4pUNyb2Ph2s%3kRS7X4ae3gFKX2TfdJ597JxobCdxeQfA3aYURdASpUmw4fTDpf3'
+YOUTUBE_API_KEY='m3DJU4hokX9haNZjvUWKkYnYUTrujAVD3FgWa9e'
+
+# Optional
+DISCORD_CSV_FILE='discord_file_from_env.csv'
+GITHUB_CSV_FILE='github_file_from_env.csv'
+GOOGLE_CSV_FILE='google_file_from_env.csv'
+REDDIT_CSV_FILE='reddit_file_from_env.csv'
+REDDIT_ENGAGEMENT_CSV_FILE='reddit_engagement_file_from_env.csv'
+STACKEXCHANGE_CSV_FILE='stackexchange_file_from_env.csv'
+STACKEXCHANGE_TOP_ANSWERERS_CSV_FILE='stackexchange_top_answerers_file_from_env.csv'
+STACKEXCHANGE_NO_ANSWERS_CSV_FILE='stackexchange_no_answers_file_from_env.csv'
+TWITTER_CSV_FILE='twitter_file_from_env.csv'
+YOUTUBE_CSV_FILE='youtube_file_from_env.csv'
+```
+
+Example `praw.ini`
+
+```
+[stats]
+bot_name=stats
+bot_version=0.0.1
+bot_author=ansible_community
+
+client_id=5SbCwr35eZNF9Q2DgDdQfA
+client_secret=fbcr3uX5s5uuwwDzwNkuJhnn3VTLDn
+
+username=ansible_community
+password=3RfwMtvWigEqkPvz
```

Comment:
Likewise here, should these be public? Or are they made up?

Reply:
also real-looking examples

Reply:
removed real-looking examples